sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ImplementationBridgeHelpers.java

```diff
@@ -1193,4 +1198,44 @@ public interface CosmosContainerIdentityAccessor {
             String getContainerLink(CosmosContainerIdentity cosmosContainerIdentity);
         }
     }
+
+    public static final class FaultInjectionRuleHelper {
+        private final static AtomicBoolean faultInjectionRuleClassLoaded = new AtomicBoolean(false);
+        private final static AtomicReference<FaultInjectionRuleAccessor> accessor = new AtomicReference<>();
+
+        private FaultInjectionRuleHelper() {
+        }
+
+        public static FaultInjectionRuleAccessor getFaultInjectionRuleAccessor() {
+            if (!faultInjectionRuleClassLoaded.get()) {
+                logger.debug("Initializing FaultInjectionRuleAccessor...");
+                initializeAllAccessors();
+            }
+
+            FaultInjectionRuleAccessor snapshot = accessor.get();
+            if (snapshot == null) {
+                logger.error("FaultInjectionRuleAccessor is not initialized yet!");
+                System.exit(9726); // Using a unique status code here to help debug the issue.
```

Comment:
Also used 9726 for diagnostics - but changed that one. So, you can continue to use 9726